tests/cli/wallet_test.py

```diff
@@ -0,0 +1,37 @@
+from cli.wallet import _send_eth, _send_skl
+from tests.constants import TEST_PK_FILE
+
+
+def test_send_eth(runner, skale):
+    address = skale.wallet.address
```

Comment:
I think we should add test to check actual amount of ETH tokens before and after sending operations for both sender and receiver accounts.